src/services/locationServiceCountries.js

```diff
@@ -0,0 +1,22 @@
+const locationServiceCountries = {
+  getCountries: async (api_key) => {
+    var headers = new Headers()
+    headers.append('X-CSCAPI-KEY', api_key)
+
+    var requestOptions = {
+      method: 'GET',
+      headers: headers,
+      redirect: 'follow'
+    }
+
+    try {
+      const response = await fetch('https://api.countrystatecity.in/v1/countries', requestOptions)
+      const result = await response.text()
+      return JSON.parse(result)
+    } catch (error) {
+      //
+    }
```

Comment:
Why we need empty `catch`? Maybe in this case we should throw another Error with intimation to the user?

Reply:
Hey Vlad, made the requested changes to the 'catch' block. Will do the same for #43 shortly. As for the tests, a separate issue #62 has been created.